src/07-projects/project-items.apib

```diff
@@ -19,6 +19,7 @@ Get a list of project items.
                     + external_cost_item
                     + time_tracking_product
             + milestone_id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string, optional)
+            + internal_cost_per_unit: null (nullable, optional)
```

Comment:
Right now the documentation is not very clear that this only accepts `null`. Should we document it as an enum type with only `null` as an option?

Reply:
I would have to document that along the lines of
```suggestion
            + internal_cost_per_unit: (enum[string], nullable, optional)
                + Members
                    + null
```

Which would suggest that the consumer should send the string 'null' and not the null value.

How about adding a comment to the filter, instead: "Only supports null as value"?